pkg/cqrs/base_cqrs/sqlc/postgres/queries.sql

```diff
@@ -447,3 +447,122 @@ FROM spans
 WHERE span_id IN (SELECT UNNEST(sqlc.slice('ids')::TEXT[]))
 LIMIT 2;
 
+-- name: GetRunSpanByRunID :one
+SELECT
+  run_id,
+  trace_id,
+  dynamic_span_id,
+  MIN(start_time) as start_time,
+  MAX(end_time) AS end_time,
```

Comment:
Are these still the start/end times? Now that we have metadata that can update things after completion, would this skew the end time?

Reply:
In all these queries, we're only checking for a single span grouped by dynamic span ID, so unless a new EXTEND span (a call to UpdateSpan) is created at the relevant level, that shouldn't be an issue.